src/groups/mqb/mqbc/mqbc_clusterdata.cpp

```diff
@@ -53,6 +53,8 @@ mqbc::ClusterDataIdentity clusterIdentity(const bslstl::StringRef& name,
     // Create client identity
     bmqp_ctrlmsg::ClientIdentity identity(allocator);
     if (!isRemote) {
+        BSLS_ASSERT_SAFE(netCluster->selfNode());
```

Comment:
On the line 65 below, `netCluster->selfNode()` is dereferenced without checks, which might lead to segfault. This might be achieved by using mock domain/cluster objects.